packages/cca/cca.0.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+authors: ["Codinuum"]
+homepage:"https://github.com/codinuum/cca/"
+maintainer: "Codinuum <codinuum@me.com>"
+bug-reports: "https://github.com/codinuum/cca/issues"
+dev-repo: "git+https://github.com/codinuum/cca.git"
+license: "Apache-2.0"
+build: [
+  [ make "-C" "src/mldiff" ]
+  [ make "-C" "src/util" ]
+  [ make "-C" "src/otreediff" ]
+  [ make "-C" "src/ast/analyzing" "production" ]
+]
+depends: [
+  "ocaml" {>= "4.11.0"}
+  "camlzip"
+  "cryptokit"
+  "csv"
+  "git-unix"
+  "menhir"
+  "ocamlnet"
+  "pxp"
+  "uuidm"
+  "volt"
```

Comment:
Could you find the correct lower-bound constraints for at least some of those (i'm thinking of at least git-unix since it API changed quite a lot of the years)

Reply:
We recently modified cca to catch up with the current (3.2.0) git-unix.

Reply:
@codinuum excellent! In that case, modify the entry above to:

```
"git-unix" {>= "3.2.0"}
```

to make sure opam selects a compatible version.